Vagrantfile

```diff
@@ -0,0 +1,26 @@
+Vagrant.configure(2) do |config|
+    config.vm.box = "fedora/25-cloud-base"
+    config.vm.network "forwarded_port", guest: 9090, host: 9090
```

Comment:
Hm, good question. I picked this one because that's where cockpit usually listens on. I'm afraid another port could confuse people, or do we have another "development port" convention?